src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -544,13 +563,29 @@ class EthService(
 
   def call(req: CallRequest): ServiceResponse[CallResponse] = {
     Future {
-      doCall(req)(ledger.simulateTransaction).map(r => CallResponse(r.vmReturnData))
+      doCall(req)(ledgerHolder().simulateTransaction).map(r => CallResponse(r.vmReturnData))
+    }
+  }
+
+  def ieleCall(req: IeleCallRequest): ServiceResponse[CallResponse] = {
```

Comment:
iele transactions return a list of values rather than a single value, so I believe you want to make a IeleCallResponse class and perform rlp decoding in this method.

Reply:
fixed